### PR TITLE
website: Update contacts for code of conduct

### PR DESCRIPTION
**Pull request by rata**
Closes: #675

### CODE REVIEW

**Comment by rata** [comment]
@russellb good point.

@daxmc99 I'm not sure your full name. I'll be away for holidays starting tomorrow, so feel free to create a new PR and close this one if I don't act. My full name is Rodrigo Campos :)

**Comment by daxmc99** [comment]
My Full name is Dax McDonald 😃

**Comment by johananl** [comment]
Force-pushed a change to add the full names (I know this is your branch @rata - doing this because of your limited availability).

**Comment by johananl** [comment]
Also - fixed a typo.

File: website/content/community/code-of-conduct.md [hunk 1]
```diff
@@ -58,11 +58,12 @@ further defined and clarified by project maintainers.
...
-complaints will be reviewed and investigated and will result in a response that
-is deemed necessary and appropriate to the circumstances. The project team is
-obligated to maintain confidentiality with regard to the reporter of an incident.
-Further details of specific enforcement policies may be posted separately.
+reported by contacting Rodrigo Campos at rodrigocc@gmail.com and Dax McDonald
+at daxmc99@gmail.com (please mail both addresses). All complaints will be
+reviewed and investigated and will result in a response that is deemed
+necessary and appropriate to the circumstances. The project team is obligated
+to maintain confidentiality with regard to the reporter of an incident. Further
+details of specific enforcement policies may be posted separately.
```
Comment (johananl) [inline website/content/community/code-of-conduct.md:66]: >Further details of specific enforcement policies may be posted separately.

I'm not sure what the purpose of this sentence is. Shouldn't we just add more details if/when we have them?

Comment (russellb) [reply website/content/community/code-of-conduct.md:66]: That's from the text of the contributor covenant, the template used here.  See the links at the bottom of the doc.

I think it's just leaving it open for the project to define some additional policies for how enforcement may be handled.  I don't think we need to do anything though.

Comment (rata) [reply website/content/community/code-of-conduct.md:66]: @russellb +1

**Review by johananl — APPROVED** [review]

**Comment by daxmc99** [comment]
Any concerns with merging this? I missed the approval

**Comment by russellb** [comment]
> Any concerns with merging this? I missed the approval

no, I think this should be merged.  I'll get it now.

**Pull request merged**